template/sample/template.gen_test.go

```diff
@@ -27,11 +27,11 @@ import (
 	"github.com/golang/protobuf/ptypes/empty"
 	rpc "github.com/googleapis/googleapis/google/rpc"
 
+	"context"
```

Comment:
this imports block is all wrong. "context" and "time" need to be up in the std lib part of the block.

Reply:
Good catch. Fixed